src/utils/validation/UnionNode.js

```diff
@@ -40,6 +40,13 @@ export default class UnionNode {
         error: 'The Union upstream nodes require attributes aligned',
       };
     }
+    if (this.upstreamAttributeNameAligned()) {
+      return {
+        isValid: false,
+        error:
+          'The Union upstream nodes require attribute to be equal to Union name',
```

Comment:
@JesusGuerrero could this error message be made more explicit/detailed? Something like:
`The Union upstream nodes require attribute to be equal to the Union name. Currently, upstream node X has attributes named A, B and C; upstream node Y has attributes Z and ...".  All these nodes should have an attribute called <Name-Of-Union-Node>"

Reply:
And this actually introduces a problem because now it is not possible for a node to participate in two union nodes. It's OK for now, but we need to work on #70 to fully address the issue consistently.